Jenkinsfile

```diff
@@ -37,19 +38,29 @@ pipeline {
     WEBHOOK_ID = 'WEBHOOK_IQ'
   }
   stages {
-    stage('Release Tag') {
+    stage('Publish Library') {
```

Comment:
I made a ticket to automate the publishing of the library when there's a change to the verison.sbt file
https://socrata.atlassian.net/browse/EN-65991